apigw-lambda-rds-snapstart/example-pattern.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "Create a REST API with Amazon API Gateway, Amazon RDS and AWS Lambda with SnapStart",
+  "description": "A pattern to show how to use Lambda SnapStart with relational databases.",
+  "language": "Java",
+  "level": "300",
+  "framework": "CDK",
+  "introBox": {
```

Comment:
I have a section in the README.md file about SnapStart best practices with RDS. Is there a way to include this content on the page as well?

Reply:
not really, people will need to read the readme, unless you have that public somewhere and you can link in the resources